doc/quickstart/configure.rst

```diff
@@ -271,192 +274,180 @@ For example, Python's ``None`` is YAML's ``null``, Python's ``True`` is YAML's
-|                               | ``log_progress_interval`` instead of   |                             |                                        |
-|                               | showing a progress bar. The value can  |                             |                                        |
-|                               | be specified in the format accepted by |                             |                                        |
-|                               | :func:`dask.utils.parse_timedelta`. A  |                             |                                        |
-|                               | negative value disables any progress   |                             |                                        |
-|                               | reporting. A progress bar is only      |                             |                                        |
-|                               | shown if ``max_parallel_tasks: 1``.    |                             |                                        |
-+-------------------------------+----------------------------------------+-----------------------------+----------------------------------------+
-
 .. _config-dask:
 
 Dask configuration
 ==================
 
+Configure Dask in the  ``dask`` section.
+
 The :ref:`preprocessor functions <preprocessor_functions>` and many of the
 :ref:`Python diagnostics in ESMValTool <esmvaltool:recipes>` make use of the
 :ref:`Iris <iris:iris_docs>` library to work with the data.
 In Iris, data can be either :ref:`real or lazy <iris:real_and_lazy_data>`.
-Lazy data is represented by `dask arrays <https://docs.dask.org/en/stable/array.html>`_.
+Lazy data is represented by `dask arrays <https://docs.dask.org/en/stable/array.html>`__.
 Dask arrays consist of many small
-`numpy arrays <https://numpy.org/doc/stable/user/absolute_beginners.html#what-is-an-array>`_
+`numpy arrays <https://numpy.org/doc/stable/user/absolute_beginners.html#what-is-an-array>`__
 (called chunks) and if possible, computations are run on those small arrays in
 parallel.
 In order to figure out what needs to be computed when, Dask makes use of a
-'`scheduler <https://docs.dask.org/en/stable/scheduling.html>`_'.
+'`scheduler <https://docs.dask.org/en/stable/scheduling.html>`__'.
 The default scheduler in Dask is rather basic, so it can only run on a single
 computer and it may not always find the optimal task scheduling solution,
 resulting in excessive memory use when using e.g. the
 :func:`esmvalcore.preprocessor.multi_model_statistics` preprocessor function.
 Therefore it is recommended that you take a moment to configure the
-`Dask distributed <https://distributed.dask.org>`_ scheduler.
+`Dask distributed <https://distributed.dask.org>`__ scheduler.
 A Dask scheduler and the 'workers' running the actual computations, are
 collectively called a 'Dask cluster'.
 
-Dask distributed configuration
-------------------------------
+Predefined Dask profiles
```

Comment:
```suggestion
Dask profiles
```

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2616/commits/a5368277069ff0cc29caafbec5ce0b864b84fbd6